components/BlogBlock.js

```diff
@@ -0,0 +1,39 @@
+import Link from 'next/link'
+import styles from '../styles/BlogBlock.module.css';
+
+
+export default function BlogBlock({ post }) {
+    return (
+
+        <div className={styles.block}>
+            <Link href={`/posts/${post.id}`}>
+                <div className={styles.blogBox}>
+                    <div className={styles.imgBox}>
+                        <Link href={`/posts/${post.id}`}><img src={post.FeaturedImage.url} className={styles.img}></img></Link>
+                    </div>
+                    <div className={styles.textBox}>
+
+                        <div>
+                            <h3>{post.Title}</h3>
+                        </div>
+                        <div>
+                            <p>{post.Date} | {post.post_category.Name}</p>
+                        </div>
+                        <div className={styles.blogDetails}>
+                            <p className={styles.content}>{post.Content}</p>
```

Comment:
The content should be truncated here as right now it will show the entire post content here.

There are a couple ways we can go about doing this, either we can just cut off the blog content at a certain number of words/characters (maybe 300?) or we could also have an option in Strapi to enter an "excerpt" this would let the user control what the blog post excerpt shown here is. (ie: here is what we have on our TTT site)
![image](https://user-images.githubusercontent.com/10680294/99283591-11d60000-27ea-11eb-81f3-5d19cd05bfb8.png)

Since it might be more work to ask Atsuko to do that, we could probably go the way of just truncating the text, or actually just chaning the design to not even show a text preview here and only show image, title, category and date posted. What do you think? I was thinking this cause if we pick an arbitrary character limit for the truncation it might end up in weird spots sometimes.

Reply:
@LRode 
![image](https://user-images.githubusercontent.com/35358725/99311754-58187700-285d-11eb-95ed-f93701c329ad.png)

This is how it looks like right now, I have it set to show only 3 lines but It can be changed. What do you think?

Reply:
With Atsuko I would try to make it as simple as possible so I wouldn't suggest doing an "excerpt". It would look better tho